src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -47,6 +47,23 @@
 logger = logging.getLogger("pyfluent.launcher")
 
 
+def _remove_unused_args(fluent_launch_mode: LaunchMode = None, **kwargs):
+
+    def _remove_key(unused_args, **kwargs):
+        for arg in unused_args:
+            if arg in kwargs:
+                kwargs.pop(arg)
+        return kwargs
+
+    if fluent_launch_mode == LaunchMode.STANDALONE or LaunchMode.PIM:
+        _remove_key(
+            ["start_container", "container_dict", "dry_run", "scheduler_options"],
+            **kwargs,
+        )
+    elif fluent_launch_mode == LaunchMode.CONTAINER:
+        _remove_key(["start_container", "scheduler_options"], **kwargs)
```

Comment:
Added to resolve codegen error.

![image](https://github.com/ansys/pyfluent/assets/106588300/11286e63-651d-4ab4-be6e-98da01c98b7a)